tests/integration/api_v2/http/transactions.test.js

```diff
@@ -110,13 +110,13 @@ describe('Transactions API', () => {
 		});
 
 		it('long transaction id -> 500', async () => {
-			const response = await api.get(`${endpoint}?id=412875216073141752800000`, 500);
+			const response = await api.get(`${endpoint}?transactionId=412875216073141752800000`, 500);
```

Comment:
The returned error code should be `400 BAD REQUEST`.

Reply:
Done!